cpp/src/text/subword/wordpiece_tokenizer.cu

```diff
@@ -414,7 +420,7 @@ wordpiece_tokenizer::wordpiece_tokenizer(hashed_vocabulary const& vocab_table,
 }
 
 uvector_pair wordpiece_tokenizer::tokenize(char const* d_strings,
-                                           uint32_t const* d_offsets,
+                                           cudf::size_type const* d_offsets,
                                            uint32_t num_strings,
```

Comment:
should this still be `uint32_t` type?
It looks like implicit conversion happens at calling code. is this not caught by warnings?